game/src/components/damage.rs

```diff
@@ -0,0 +1,31 @@
+use bevy::prelude::*;
+
+///Pointer component to mark entities that need a Damage calculation
+#[derive(Component, Debug)]
+pub struct Damage {
+    pub damage_value: u32,
+}
+
+impl Damage {
+    pub fn new(damage_value: u32) -> Self {
+        Self { damage_value }
+    }
+}
+
+///Component to represent how much damage an entity does
+#[derive(Component, Debug)]
+pub struct DamageApplied {
```

Comment:
Rename to something more descriptive. The naming currently seems to suggests that the entity holding it has damage applied to it.

Reply:
completed